tasks/src/main/java/taf/product/google/cloud/util/StringUtils.java

```diff
@@ -0,0 +1,27 @@
+package taf.product.google.cloud.util;
+
+public class StringUtils {
```

Comment:
The class doesn't look like an util one, it just stores constants and does nothing. DataStorage probably a better name